modules/avatar/react/lib/Avatar.tsx

```diff
@@ -35,6 +35,11 @@ export interface AvatarProps extends React.ButtonHTMLAttributes<HTMLButtonElemen
    * Will render an `div` tag instead of a `button` when defined.
    */
   as?: 'div';
+  /**
+   * If true, render the image and hide the Avatar background.
+   * @default false
+   */
+  imageLoaded?: boolean;
```

Comment:
We don't want to add `imageLoaded` to the `AvatarProps` interface. It is only used by the `StyledContainer` and will be ignored by `Avatar`.